tee_launcher/pytest.ini

```
@@ -0,0 +1,6 @@
+[pytest]
+markers =
+    slow: marks tests as slow (deselect with '-m "not slow"')
+    ci_excluded: marks tests that should be excluded from ci
+    no_atexit_cleanup: Skip the autouse atexit cleanup fixture for this test
```

```suggestion
    ci_excluded: marks tests that should be excluded from ci
```

As Reynaldo said, the `slow` and `no_atexit_cleanup` flags are unused. Let's remove them.

---

if test `test_validate_image_hash_stress` is removed as suggested, even `ci_excluded` will not be used, so the file itself can be removed